tools/tests/com.liferay.ide.project.ui.tests/src/com/liferay/ide/project/ui/tests/page/SelectPortletFrameworkPageObject.java

```diff
@@ -1,120 +1,120 @@
-    private RadioPageObject<SWTBot> jsfRadio;
-    private RadioPageObject<SWTBot> liferayMVCRadio;
-    private RadioPageObject<SWTBot> portletFrameworkRadio;
-    private LabelPageObject<SWTBot> portletNameLabel;
-    private TextPageObject<SWTBot> portletNameText;
-    private RadioPageObject<SWTBot> springMVCRadio;
-    private RadioPageObject<SWTBot> vaadinRadio;
-    public SelectPortletFrameworkPageObject( T bot )
-    {
-        this( bot, TEXT_BLANK, INDEX_DEFAULT_VALIDATION_MESSAGE );
-    }
-
-    public SelectPortletFrameworkPageObject( T bot, String title )
-    {
-        this( bot, title, INDEX_DEFAULT_VALIDATION_MESSAGE );
-    }
-
-    public SelectPortletFrameworkPageObject( T bot, String title, int indexValidationMessage )
-    {
-        super( bot, title, BUTTON_BACK, BUTTON_NEXT, BUTTON_FINISH, BUTTON_CANCEL );
-
-        liferayMVCRadio = new RadioPageObject<SWTBot>( bot, TEXT_LIFERAY_MVC_FRAMEWORK );
-        jsfRadio = new RadioPageObject<SWTBot>( bot, TEXT_JSF_FRAMEWORK );
-        springMVCRadio = new RadioPageObject<SWTBot>( bot, TEXT_SPRING_MVC_FRAMEWORK );
-        vaadinRadio = new RadioPageObject<SWTBot>( bot, TEXT_VAADIN_FRAMEWORK );
-        portletNameText = new TextPageObject<SWTBot>( bot, LABEL_PORTLET_NAME );
-        displayNameText = new TextPageObject<SWTBot>( bot, LABEL_DISPLAY_NAME );
-        portletNameLabel = new LabelPageObject<SWTBot>( bot, LABEL_PORTLET_NAME );
-        displayNameLabel = new LabelPageObject<SWTBot>( bot, LABEL_DISPLAY_NAME );
-    }
-
-    public SelectPortletFrameworkPageObject( T bot, String title, String radio )
-    {
-        this( bot, title );
-
-        selectFramework( radio );
-    }
-
-    public boolean IsLiferayMVCRadioSelected()
-    {
-        return liferayMVCRadio.isSelected();
-    }
-
-    public boolean isVisibleProjectNameAndDisplayName()
-    {
-
-        if( portletNameLabel.isVisible( LABEL_PORTLET_NAME ) && displayNameLabel.isVisible( LABEL_DISPLAY_NAME ) )
-        {
-            return true;
-        }
-        else
-        {
-            return false;
-        }
-
-    }
-
-    public void selectFramework( String radio )
-    {
-        if( radio.equals( liferayMVCRadio.getLabel() ) )
-        {
-            liferayMVCRadio.click();
-        }
-        else if( radio.equals( jsfRadio.getLabel() ) )
-        {
-            jsfRadio.click();
-        }
-        else if( radio.equals( springMVCRadio.getLabel() ) )
-        {
-            springMVCRadio.click();
-        }
-        else if( radio.equals( vaadinRadio.getLabel() ) )
-        {
-            vaadinRadio.click();
-        }
-    }
-
-    public void setProjectNameAndDisplayName( String projectName, String displayName )
-    {
-        if( isVisibleProjectNameAndDisplayName() )
-        {
-            portletNameText.setText( projectName );
-            displayNameText.setText( displayName );
-        }
-    }
-}
+/*******************************************************************************

```

Comment:
THis class doesn't look like it has changed any, it should not be shown as every line different.